### PR TITLE
payment pages and registration updates

### PR DESCRIPTION
**Pull request by cassandradamico**
Adding content in payment pages
tried to add a grid in some payment pages. it shows error due to format (although offered in the documentation)
some pages do accept, some others not the --- at the footer for the Support contact

Registration pages content slightly updates

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -66,28 +65,27 @@ nav:
...
-      - 'Update PA payment status': 'payment/change-pa-status-program.md'
-      - 'Import payment instructions in FSP portal': 'payment/issue-payment-fsp-portal.md'
-      - 'Messages in 121 platform': 'payment/list-messages-payments.md'
-      - 'Issue payments': 'payment/issue-payment-integrated-fsp.md'
-      - 'Send and reconcile payment instructions (FSP portal)': 'payment/issue-payment-fsp-portal.md'
-      - 'Notification messages to 121 payment users': 'payment/list-messages-payments.md'
+      - 'Payment page': 'payment/payment.md'
+      - 'Issue instant payments': 'payment/issue-payment-integrated-fsp.md'
+      - 'Send and reconcile manual payment instructions (FSP portal)': 'payment/issue-payment-fsp-portal.md'
+      - 'Pause payments sent to PAs': 'payment/pause-payment-pa.md'
```
Comment (tijsziere) [inline mkdocs.yml:84]: This looks like you can pause payments that you're sending to PAs, we cannot do that, once a payment is executed you cannot revert it

File: mkdocs.yml [hunk 2]
```diff
@@ -66,28 +65,27 @@ nav:
...
+      - 'Payment statuses in 121': 'payment/list-status-payment-page.md'
+      - 'Payment reporting and activty': 'payment/payment-reporting.md'
     - 'Evaluation':
```
Comment (tijsziere) [inline mkdocs.yml:87]: We don't have the evaluation page anymore

File: docs/payment/issue-payment-fsp-portal.md [hunk 3]
```diff
@@ -4,41 +4,94 @@ hide:
...
+- **Apply Action** button becomes blue and clickable. Click the button to start paying out to the selected PA.
+- A pop-up appears to confirm the sum to be transferred value to each household and the **Maximum amount that will be spend this payment: SUM.**, matching the total sum of the transfers.
+- Click on **Start paying now**.
+- A message indicates the completion of the action and **next steps to download the payment instructions**.
 
-Select beneficiaries. Bulk or individual.
+Only PAs who have not received the **payment#** round are available for selection. PA’s that have received the max payments will not be applicable (max payment number per household might change as the program continues).
 
-### Export payment instructions
+### Export the payment instructions
```
Comment (tijsziere) [inline docs/payment/issue-payment-fsp-portal.md:35]: I would actually remove 'the'

File: docs/registration/registration-kobo-form.md [hunk 4]
```diff
@@ -1,16 +1,35 @@
...
+  - toc
 ---
 
-!!! info "This section explains"
-    Steps to link your kobo form with the 121 platform
+Unless you manually import registrations with an excel .CSV file, registrations done with KOBO will automatically be imported within your program in 121 Platform, in the **People Affected** page.
 
-- Create your kobo form on the [IFRC server](https://kobo.ifrc.org/)
-- Share this koboform with account `nlrc121platform`
+- Create your Kobo-form on the KOBO tool
```
Comment (tijsziere) [inline docs/registration/registration-kobo-form.md:9]: we need to be consistent with Kobo - KOBO - ???

**Review by tijsziere — CHANGES_REQUESTED** [review]

File: mkdocs.yml (hunk 2 above)
Comment (tijsziere) [reply mkdocs.yml:87]: In the portal I mean, we got rid of that page when we introduced the 'people affected' page

File: mkdocs.yml (hunk 1 above)
Comment (cassandradamico) [reply mkdocs.yml:84]: @emily-walkin do you have a way to phrase that better?

File: mkdocs.yml (hunk 2 above)
Comment (cassandradamico) [reply mkdocs.yml:87]: @tijsziere do we still have a PDM in 121? I am not sure to understand

File: docs/payment/payment.md [hunk 5]
```diff
@@ -3,40 +3,63 @@ title: Program overview - payment
 hide:
   - toc
 ---
+The Payment page enables to oversee the payment status for all included beneficiaries in the program and execute payments.
```
Comment (emily-walkin) [inline docs/payment/payment.md:6]: The Payment page **allows you** to oversee the payment status

File: docs/payment/payment.md [hunk 6]
```diff
@@ -3,40 +3,63 @@ title: Program overview - payment
...
-### Overview of pending payment
+### Process and execute payments to PA
 
-### Issue and Review payments to beneficiaries
+In the 121 Platform, payments can be processed with two different methods:
 
-Payment rounds
-Indicative messages (successful, waiting, refused)
-Adjust payment methods and bank details whenever an error is made.
+- **Send and Reconcile manual payments by exporting a payment instructions file**. You pre-process the payment within 121 by including the selected PA in the payment round, then download the payment instructions to be sent to the FSP and later reconciliation the payment data. Further instructions on [this page](./issue-payment-fsp-portal.md)
```
Comment (emily-walkin) [inline docs/payment/payment.md:16]: ...and later **reconcile** the payment data.

File: docs/payment/payment.md [hunk 7]
```diff
@@ -3,40 +3,63 @@ title: Program overview - payment
...
+### Overview of payment status
 
-!!! info "Integration with FSP and aggregator to 121"
-    121 platform has integrated with a certain number of local FSP and aggregators. Find the list of existing integration here.
-    Additional local FSP can be integrated upon evaluation. Please, share your request with our team.
+In the **Payment** page, you will find the overview of included beneficiaries in the program. Depending on the program phases, PAs can have different statuses.
+The overview is particularly interesting for:
 
-### Export and reconcile payment instructions
+- Finance Manager and Finance Officer to keep tracked of transactions statuses, successful and failed ones.
```
Comment (emily-walkin) [inline docs/payment/payment.md:25]: keep **track** of **payment** statuses,

File: docs/payment/payment.md [hunk 8]
```diff
@@ -3,40 +3,63 @@ title: Program overview - payment
...
+- CVA Manager and CVA Officer to keep informed and follow wether or not a PA is receiving cash support.
```
Comment (emily-walkin) [inline docs/payment/payment.md:26]: **whether**

File: mkdocs.yml (hunk 1 above)
Comment (emily-walkin) [reply mkdocs.yml:84]: Pause the inclusion of PAs in payment rounds

**Review by emily-walkin — APPROVED** [review]

File: docs/payment/issue-payment-fsp-portal.md (hunk 3 above)
Comment (tijsziere) [reply docs/payment/issue-payment-fsp-portal.md:35]: @cassandradamico why did you resolve this? I don't see it updated in the branch?

Comment (cassandradamico) [reply docs/payment/issue-payment-fsp-portal.md:35]: @tijsziere it is corrected on my side, weird

Comment (tijsziere) [reply docs/payment/issue-payment-fsp-portal.md:35]: @cassandradamico maybe commit your latest changes? Have you saved all your files?

**Review by tijsziere — APPROVED** [review]

**Pull request merged**